### PR TITLE
style(frontend): change titles and images of about modals

### PR DESCRIPTION
**Pull request by AntonioVentilii-DFINITY**
# Motivation

Updating titles and images of about modals.

### CODE REVIEW

File: src/frontend/src/lib/components/hero/about/AboutWhatModal.svelte [hunk 1]
```diff
@@ -23,18 +22,12 @@
...
-			<IconIncognito />
-			<Html text={replaceOisyPlaceholders($i18n.about.what.text.go_incognito)} />
-		</p>
-
 		<p class="mt-6">
 			<IconWalletConnect />
 			<Html text={replaceOisyPlaceholders($i18n.about.what.text.use_eth_dapps)} />
 		</p>
 
-		<!-- TODO: Remove the opacity when it will be possible to use IC dapps with Oisy-->
```
Comment (peterpeterparker) [inline src/frontend/src/lib/components/hero/about/AboutWhatModal.svelte:36]: Removing this TODO is related to this PR?

**Review by peterpeterparker — COMMENTED** [review]
The new `cover-features.png` image has rounded corners, is that expected?

File: src/frontend/src/lib/components/hero/about/AboutWhatModal.svelte (hunk 1 above)
Comment (AntonioVentilii-DFINITY) [reply src/frontend/src/lib/components/hero/about/AboutWhatModal.svelte:36]: Correct, they changed idea: we can already remove the opacity, independently of having already the feature or not

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

**Pull request merged**